### ISSUE TITLE
Set does not function in 5.1.0 release

### ISSUE THREAD

**Issue by tomporter518** [codetheweb/tuyapi#217]
**Describe the bug**
After updating to tuyapi 5.1.0, .set() no longer functions. The switch is not controlled AND no data is returned via the on 'data' handler. Debug output has no error or further detail.

**To Reproduce**
1. Update to package version 5.1.0
2. Run asynchronous sample for a device

**Expected behavior**
Switch to be controlled as requested by the .set() method.

**Debug Output**
```
  TuyAPI IP and ID are already both resolved. +0ms
  TuyAPI Connecting to 10.5.18.YY... +6ms
  TuyAPI Socket connected. +94ms
Connected to device!
  TuyAPI GET Payload: +3ms
  TuyAPI { gwId: '02200370dc4f2219XXXX', devId: '02200370dc4f2219XXXX' } +0ms
  TuyAPI Received data: 000055aa000000010000000a00000044000000007b226465764964223a223032323030333730646334663232313966643735222c22647073223a7b2231223a66616c73652c2232223a307d7d490b4b780000aa55 +19ms
  TuyAPI Parsed: +5ms
  TuyAPI { payload:
  TuyAPI    { devId: '02200370dc4f2219XXXX', dps: { '1': false, '2': 0 } },
  TuyAPI   leftover: false,
  TuyAPI   commandByte: 10,
  TuyAPI   sequenceN: 1 } +0ms
Data from device: { devId: '02200370dc4f2219XXXX', dps: { '1': false, '2': 0 } }
Boolean status of default property: false.
  TuyAPI SET Payload: +2ms
  TuyAPI { devId: '02200370dc4f2219XXXX',
  TuyAPI   gwId: '02200370dc4f2219XXXX',
  TuyAPI   uid: '',
  TuyAPI   t: 1560714843,
  TuyAPI   dps: { '1': true } } +0ms
  TuyAPI Disconnect +10s
  TuyAPI Socket closed: 10.5.18.YY +2ms
Disconnected from device.
```

**Desktop (please complete the following information):**
 - OS: Fedora
 - OS: Version 30
 - Node Version: v10.16.0

**Additional context**
Reverting to 5.0.3 restores functionality.

**Comment by kueblc** [comment]
Thanks for reporting, I'm looking into this

**Comment by kueblc** [comment]
Apologies for any inconvenience, and thank you again for the prompt report. Will be fixed in v5.1.1 per #218.

**Comment by tomporter518** [comment]
Thanks for the fix and continued work. I have tested and everything works as expected in 5.1.1.
